src/controllers/authenticationController.ts

```diff
@@ -118,6 +121,41 @@ export default class AuthenticationController {
 
     const spidUser = errorOrSpidUser.right;
 
+    // check for CIE IDP test login
+    const maybeCieTestUserInWhitelist = pipe(
+      Object.keys(CIE_IDP_IDENTIFIERS).filter(
+        identifier =>
+          identifier !==
+          "https://idserver.servizicie.interno.gov.it/idp/profile/SAML2/POST/SSO"
```

Comment:
Wouldn't be better to put this URL is some config?

Reply:
this URL is already defined in `CIE_IDP_IDENTIFIERS` array, inside`io-spid-commons`. since positions can change/new elements can be added i decided to leave this url as you see. do you have some better option?

Reply:
And what about `${"if a CIE TEST user is in whitelist"} | ${validUserPayload.fiscalNumber} | ${Object.keys(CIE_IDP_IDENTIFIERS)[0]}` in tests then, will it never change position? 

I think we should find a more generic way to bind an environment identifier to its url. If position in array can change why not refactoring to an object such this?
```
{
   CIE_TEST: "<url_test>",
   CIE_PROD: "<url_prod>"
}
```

I'm a bit out of context about what there is in `io-spid-commons` and the reason why it is like that, but I think that we should have not rely on the position in the array of an element, at least we should use a tuple so that order will never change. 

Hardcoded urls are more difficult to update in the future than an env var, in my opinion. This PR is going to work anyway, but we should just be conscious that any future change to this url will be also in other points of the code.

cc @BurnedMarshal

Reply:
I was wrong in the comment. the `CIE_IDP_IDENTIFIERS` is an object containing the identifiers given to specific URLS. that explains the presence of `Object.keys(CIE_IDP_IDENTIFIERS)` there. CIE test environments can be added in the future so the simplest solution would be to add an object like you said inside `io-spid-commons` containing arrays of CIE environments (prod and tests). so in the future you would need to edit 2 variables in the same file in `io-spid-commons`.

Reply:
...or using just some comma separated values we could just use an object that point to 2 configuration variables and we could change them whenever we want.

By the way. You can go. I think it's not in scope of this PR. 👍

Reply:
The same for https://github.com/pagopa/io-backend/pull/967/files#r1094404637, why not using index also here? I know that an order change would break the code, but I think also that it should happen. As we could not trust indexes, we can't trust the string neither? What if the prod url change in commons? We should upgrade it also here. So I'd comment on commons to not change the order because those url are used by index.